shell/platform/tizen/public/flutter_platform_view.h

```diff
@@ -16,8 +16,9 @@ using ByteMessage = std::vector<uint8_t>;
 
 class PlatformView {
  public:
-  PlatformView(flutter::PluginRegistrar* registrar, int viewId)
-      : registrar_(registrar),
+  PlatformView(flutter::PluginRegistrar* registrar, int viewId, void* win)
+      : win_(win),
```

Comment:
Against the naming rule.

> Do not abbreviate by deleting letters within a word.

I recommend you to read the [style guide](https://google.github.io/styleguide/cppguide.html) from end to end, or at least the "Background" and "Naming" sections.

Reply:
It's fixed now!